concent_api/core/tests/test_unit_authentication_utilities.py

```diff
@@ -136,45 +136,45 @@ def test_validate_golem_message_signed_with_key_should_raise_error_if_incorrect_
 
 class ValidateListOfIdenticalTaskToComputeUnitTest(TestCase):
```

Comment:
BTW: I think ot should inherit from `unittest.TestCase`. Inheriting from `django.test.TestCase` seems a bit of overkill for simple unit test ;)

Reply:
Done